### DISCUSSION TITLE
[WARNING] [publicarray-au2] Incorrect signature for provider name: [2.dnscrypt-cert.dns.seby.io.]

### DISCUSSION THREAD

**Discussion by MojtabaFaal** [DNSCrypt/dnscrypt-proxy#1799] [Q&A] [answered]
Hi
What is this warning?

`[2021-07-28 22:57:33] [WARNING] [publicarray-au2] Incorrect signature for provider name: [2.dnscrypt-cert.dns.seby.io.]`

I receive this warning yesterday and today
Is it a dangerous subject?
What should I do ?

**Comment by jedisct1** [comment, marked as answer]
Hi,

It means that there was a configuration error on the server. Nothing to worry about on your side, and it looks like this has been fixed :)

**Reply by MojtabaFaal** [reply]
Hi
The warning is still there

`[2021-07-31 14:27:19] [WARNING] [publicarray-au2] Incorrect signature for provider name: [2.dnscrypt-cert.dns.seby.io.]`

**Reply by jedisct1** [reply]
Confirmed here:

```sh
doggo example.com \
@sdns://AQcAAAAAAAAAEjEzOS45OS4yMjIuNzI6ODQ0MyB3NPVeil3tZfA7hH0fz-MBp7VSd6pprpccj78c5dOjFRsyLmRuc2NyeXB0LWNlcnQuZG5zLnNlYnkuaW8
```

```text
2021/07/31 13:27:13 [2.dnscrypt-cert.dns.seby.io.] incorrect signature
ERROR[2021-07-31T13:27:13+02:00] error loading resolver                        error="no useable certificate found"
```

@publicarray can you take a look? Is the stamp not up to date?

**Reply by publicarray** [reply]
Thanks @jedisct1 Yea looks like I've done it again, sorry. Updating the container lost the keys again. The new stamp is: `sdns://AQcAAAAAAAAAEjEzOS45OS4yMjIuNzI6ODQ0MyDR7bj6zoAmbRaE1B8qTkCL_O84QCDMYPUgXZy5FRqUYRsyLmRuc2NyeXB0LWNlcnQuZG5zLnNlYnkuaW8`

**Reply by MojtabaFaal** [reply]
Right now: (Time zone: Tehran, Iran)

`[2021-07-31 14:27:19] [WARNING] [publicarray-au2] Incorrect signature for provider name: [2.dnscrypt-cert.dns.seby.io.]`

`[2021-07-31 18:32:18] [WARNING] [publicarray-au2] Incorrect signature for provider name: [2.dnscrypt-cert.dns.seby.io.]`

`[2021-07-31 20:09:51] [WARNING] [publicarray-au2] Incorrect signature for provider name: [2.dnscrypt-cert.dns.seby.io.]`

**Reply by jedisct1** [reply]
Stamp updated in the lists.

**Reply by MojtabaFaal** [reply]
Question: If the problem is solved, then why is the alert message displayed to me?

Right now: (Time zone: Tehran, Iran)

`[2021-08-01 15:27:36] [WARNING] [publicarray-au2] Incorrect signature for provider name: [2.dnscrypt-cert.dns.seby.io.]`

**Reply by jedisct1** [reply]
Your `public-resolvers.md` file is probably outdated. It will eventually be automatically updated. You can also just delete it and restart the proxy.

**Reply by MojtabaFaal** [reply]
Yes
The problem was fixed
Thanks to @jedisct1 & @publicarray
